### PR TITLE
Dashboard Widget

### PR DESCRIPTION
**Pull request by tomauer**
Work In Progress! No mergy, yet.

### CODE REVIEW

File: public/js/views/lists/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    
+    setup: function () {    
+      return Views.ListView.prototype.setup.call(this);
+    },
+    
+    initDialog: function() {
+        console.log("init");
+        console.log(CartoDB);
+        
+        //WHEN IS CartoDB available?
```
Comment (tomauer) [inline public/js/views/lists/dashboard.js:74]: @sanderpick When is the CartoDB object available? I'm trying to pull in the dashboard data as soon as it initializes, but when I call this from the initialize function, it never has the sql and url objects assigned to it yet.

**Comment by tomauer** [comment]
@sanderpick if you have a chance, I have a few questions about the framework and how to get a list view active. I can't seem to register it in a way that makes sense and has the initialize function called on it. Trying to emulate the species list view, but can't seem to link it with its parent.

**Comment by sanderpick** [comment]
Will take a look tomorrow, @tomauer!

**Comment by sanderpick** [comment]
Hey @tomauer, I'm failing to keep up o this ... my apologies. What's your status with this?

**Comment by tomauer** [comment]
Still looking for some guidance on how to initialize a list view without adding it as a map control and wanting to know when the CartoDB object is available. Take a look at the dashboard classes that I've added.

**Pull request closed without merging**